tests/test_p4objects.cpp

```diff
@@ -517,6 +517,12 @@ class JsonBuilder {
     Json::Value cond(Json::objectValue);
     cond["name"] = name;
     cond["id"] = 0;
+    Json::Value source_info(Json::objectValue);
```

Comment:
Sounds good.  Thanks for all the help in getting these changes into committable shape.